py-polars/polars/expr/string.py

```diff
@@ -379,60 +388,65 @@ def lengths(self) -> Expr:
 
         Examples
         --------
-        >>> df = pl.DataFrame({"s": ["Café", None, "345", "東京"]}).with_columns(
-        ...     [
-        ...         pl.col("s").str.lengths().alias("length"),
-        ...         pl.col("s").str.n_chars().alias("nchars"),
-        ...     ]
+        >>> df = pl.DataFrame({"a": ["Café", "345", "東京", None]})
+        >>> df.with_columns(
+        ...     pl.col("a").str.len_bytes().alias("bytes"),
+        ...     pl.col("a").str.len_chars().alias("chars"),
         ... )
-        >>> df
         shape: (4, 3)
-        ┌──────┬────────┬────────┐
-        │ s    ┆ length ┆ nchars │
-        │ ---  ┆ ---    ┆ ---    │
-        │ str  ┆ u32    ┆ u32    │
-        ╞══════╪════════╪════════╡
-        │ Café ┆ 5      ┆ 4      │
-        │ null ┆ null   ┆ null   │
-        │ 345  ┆ 3      ┆ 3      │
-        │ 東京  ┆ 6      ┆ 2      │
-        └──────┴────────┴────────┘
+        ┌──────┬───────┬───────┐
+        │ a    ┆ bytes ┆ chars │
```

Comment:
Suggest example column names of `nchars` and `nbytes` instead 👍

Reply:
Hope you will accept `n_chars` and `n_bytes` 😄

Reply:
Even better ;)